### DISCUSSION TITLE
Table Header Overlaps V2.52

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#2104] [Q&A] [answered]
Hi

I have tried to reproduce this here.

https://play.tailwindcss.com/SvIt6zkCkE

This is what shows on my side. How do I fix the header columns so that they dont overlap (some headings are cut off).

I also want to header text to start from the bottom of the header and not at the center.

![image](https://github.com/saadeghi/daisyui/assets/5799306/59e504b6-5bde-4d09-bc7e-bac1cb0ad013)

Can you please advise? Thanks.

**Comment by saadeghi** [comment, marked as answer]
I think you're not using the latest version of daisyUI.  
Upgrade to the latest version with `npm i -D daisyui@latest`  
Here are the [changelog](https://daisyui.com/docs/changelog/#300-2023-06-01) and [breaking changes](https://v2.daisyui.com/docs/changelog/#breaking-changes) for v2 to v3

Also, for rotating the text in table, I wouldn't suggest CSS rotate because each column has different width and when you use rotate, the cell height should fit in the height and the height should fit the width and it's not easy to put them together. Use [CSS `writing-mode`](https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode) instead. Tailwind class name for that would be `[writing-mode:vertical-lr]`
